docker/README.md

```diff
@@ -255,21 +255,19 @@ The predict endpoint will return a prediction response in JSON. It will look som
 
 ## Advanced Settings
 
-#### Description of Settings in mms_app.conf
+#### Description of Settings in mms_app_cpu.conf
 
-The system settings are stored in `mms_docker_cpu/mms_app.conf`. You can modify these settings to use different models, or to apply other customized settings. The default settings were optimized for a C4.8xlarge instance.
+The system settings are stored in [mms_app_cpu.conf](./mms_app_cpu.conf). You can modify these settings to use different models, or to apply other customized settings. The default settings were optimized for a C5.2xlarge instance.
 
 Notes on a couple of the parameters:
 
-* **models** - the model used when setting up service. By default it uses resnet-18, chnage this argument to use customized model.
-* **worker-class** - the type of Gunicorn worker processes. We configure by default to gevent which is a type of async worker process. Options are [described in the Gunicorn docs](http://docs.gunicorn.org/en/stable/settings.html#worker-class).
+* **models** - the model used when setting up service. By default it uses resnet-18, change this argument to use customized model.
+* **worker-class** - the type of Gunicorn worker processes. We configure by default to gevent which is a type of async worker process. Options are [described in the Gunicorn docs](http://docs.gunicorn.org/en/stable/settings.html#worker-class). We recommend setting number of workers equal to number of vCPUs in the instance you are using. A detailed discussion of experiments and results can be found [here](../docs/optimised_config.md##number-of-gunicorn-workers(workers))
```

Comment:
This is talking about worker-class. You might want to add a new highlight for "num-workers".